src/main/resources/application.properties

```diff
@@ -57,3 +57,4 @@ google.projectCreateTimeoutSeconds=600
 google.coreBillingAccount=00708C-45D19D-27AAFA
 google.parentResourceType=folder
 google.parentResourceId=270278425081
+google.singleDataProjectId=${GOOGLE_CLOUD_DATA_PROJECT}
```

Comment:
Will it ever matter if this is out of sync with the billing account? I am slightly concerned that I could charge the wrong group now that this is corrected. (Which will be my own fault, not this prs, lol)